package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "description": "Mozilla in-place localization tool.",
   "private": true,
-  "dependencies": {},
+  "dependencies": {
+    "node-pre-gyp": "^0.6.39"
```

Comment:
not sure if all these explicit dependencies are actually required - npm grumbles about dep issues - but this doesnt seem to help